hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp

```diff
@@ -88,25 +88,25 @@
             <table class="dl-table table">
               <thead>
                 <tr rsf:id="header:">
-                  <th class="flc-pager-sort-header" rsf:id="name"><a href="javascript:;">Name</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="status"><a href="javascript:;">Status</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="start"><a href="javascript:;">Start</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="end"><a href="javascript:;">End</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="total"><a href="javascript:;">Total</a></th>
+                  <th scope="col" class="flc-pager-sort-header" rsf:id="name"><a href="#">Name</a></th>
```

Comment:
not actually sure if <a> tag does.
it was original skip reading the table also all the content in the table.
research from:

*http://webaim.org/techniques/tables/
<http://webaim.org/techniques/tables/>*
*http://www.w3.org/WAI/tutorials/tables/
<http://www.w3.org/WAI/tutorials/tables/>*
*http://www.w3.org/TR/html4/struct/tables.html#h-11.2.1
<http://www.w3.org/TR/html4/struct/tables.html#h-11.2.1>*


On Thu, Aug 13, 2015 at 11:30 AM, Tim Levett <notifications@github.com>
wrote:

> In hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp
> <https://github.com/UW-Madison-DoIT/hrs-portlets/pull/38#discussion_r36994811>
> :
>
> > @@ -88,25 +88,25 @@
> >              <table class="dl-table table">
> >                <thead>
> >                  <tr rsf:id="header:">
> > -                  <th class="flc-pager-sort-header" rsf:id="name"><a href="javascript:;">Name</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="status"><a href="javascript:;">Status</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="start"><a href="javascript:;">Start</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="end"><a href="javascript:;">End</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="total"><a href="javascript:;">Total</a></th>
> > +                  <th scope="col" class="flc-pager-sort-header" rsf:id="name"><a href="#">Name</a></th>
>
> Does having the <a> in there break the sorting mechanism that is in fluid
> sorting?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Madison-DoIT/hrs-portlets/pull/38/files#r36994811>.
>